AxonDeepSeg/models/download_model.py

```diff
@@ -1,8 +1,17 @@
-from AxonDeepSeg.ads_utils import download_data
+from AxonDeepSeg.ads_utils import *
+from pathlib import Path
 import shutil
 
 
-def main():
+def download_model(destination = None):
+    if destination is None:
+        sem_destination = Path("AxonDeepSeg/models/default_SEM_model")
+        tem_destination = Path("AxonDeepSeg/models/default_TEM_model")
+    else:
+        destination = convert_path(destination)
+        sem_destination = destination / "default_SEM_model"
```

Comment:
I was unable to test the optional destination input. I don't think the path can be passed in arguments to `download_models` in command line from the entry point in `setup.py`, only if the function download_model() is called directly (like in the tests).

Reply:
@mariehbourget yes you're right, I haven't implemented that feature for the CLI tool - only the within python. I needed to do this to be able to test it separately from the mode installed during download. I could implement it for the CLI now, or to merge this issue quicker we could open a new issue for that new feature.

Reply:
Ok, I suggest we merge now and open a new issue. I'll approve the PR.